concepts/observability-strategy/strategy.md

```diff
@@ -57,12 +63,12 @@ To sum it up, the goals of Kyma observability should be:
 
 ## Architecture
 
-The proposal introduces a new preconfigured agent layer that's responsible for collecting all telemetry data. Hereby, the collection is very dependent on the signal type. The agents can be configured at runtime with different signal pipelines with basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals through the pipelines to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The users will still be able to install them manually with a blueprint, but they will no longer be part of the Kyma offering.
+A new preconfigured agent layer gets introduced which is responsible for collecting all telemetry data without being bound to any backend. Hereby, the collection is very dependent on the signal type. The agents can be configured at runtime with different signal pipelines with basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals through the pipelines to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The users will still be able to install them manually with a blueprint, but they will no longer be part of the Kyma offering.
```

Comment:
```suggestion
The proposal introduces a new preconfigured agent layer that's not bound to any backend. The agent layer is responsible for collecting all telemetry data, depending on the signal type.

Users can configure the agents at runtime with different signal pipelines using basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals through the pipelines to the configured backends. The dynamic configuration and management of the collector is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`.

The existing Kyma backends and UIs will be just one possible solution to integrate with. The users will still be able to install them manually with a blueprint, but they will no longer be part of the Kyma offering.
```